tests/unit/test_file.py

```diff
@@ -428,7 +428,7 @@ def setUp(self):
 
     def tearDown(self):
         if os.path.exists(self.path):
```

Comment:
Minor: this if statement is already checked within remove_test_file(self.path) so it can be removed.